pkg/registry/client.go

```diff
@@ -22,7 +23,9 @@ func (t *Client) Fetch(keychain authn.Keychain, repoName string) (v1.Image, stri
 
 	image, err := remote.Image(reference, remote.WithAuthFromKeychain(keychain))
 	if err != nil {
-		return nil, "", err
+		return nil, "", &reconciler.NetworkError{
```

Comment:
should this be returned for writes too?

Reply:
Well, actually the `RegistryClient.Save()` method is been [called](https://github.com/pivotal/kpack/blob/44257a70c2c9e9ac5703eadc871c7c9be22cfadc/pkg/cnb/create_builder.go#L74) in the same logic that is been used by the reconciler. So, my answer would be yes!

Reply:
@tylerphelan I just added the same error handling method to the Save operation